From 3e5df432efa29bc808b348538f657cbbef6ab4e5 Mon Sep 17 00:00:00 2001
From: Nehal Patel <nehal@hpe.com>
Date: Thu, 5 Nov 2015 22:23:06 -0800
Subject: [PATCH 13/22] Add Resilient ECMP CLI

Add the hooks in switchd to get Resilient ECMP
flag updates and pass the params down through
to the platform plugin

CLI:
[no] ip ecmp load-balance resilient disable

Change-Id: Ie12c34261af39f90a0b81f8066eeac8cecbd598c
Signed-off-by: Nehal Patel <nehal@hpe.com>

%% original patch: 0015-Add-Resilient-ECMP-CLI.patch
---
 ofproto/ofproto.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ofproto/ofproto.h b/ofproto/ofproto.h
index a356470..53aa127 100644
--- a/ofproto/ofproto.h
+++ b/ofproto/ofproto.h
@@ -267,6 +267,7 @@ struct ofproto_route {
 #define OFPROTO_ECMP_HASH_DSTPORT        0x2     /* source L4 port */
 #define OFPROTO_ECMP_HASH_SRCIP          0x4     /* source IP v4/v6 */
 #define OFPROTO_ECMP_HASH_DSTIP          0x8     /* source IP v4/v6 */
+#define OFPROTO_ECMP_HASH_RESILIENT      0x10    /* resilient hashing */
 
 enum ofproto_host_action {
     OFPROTO_HOST_ADD,
-- 
1.9.1

